client/extensions/woocommerce/app/reviews/reviews-filter-nav.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getReviewsCurrentSearch } from 'woocommerce/state/ui/reviews/selectors';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import Search from 'components/search';
+import SectionNav from 'components/section-nav';
+import { updateCurrentReviewsQuery } from 'woocommerce/state/ui/reviews/actions';
+
+class ReviewsFilterNav extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			ID: PropTypes.number,
+		} ),
+		status: PropTypes.string,
+		search: PropTypes.string,
+	};
+
+	doSearch = ( search ) => {
+		this.props.updateCurrentReviewsQuery( this.props.site.ID, { search, status: 'any' } );
+	}
+
+	clearSearch = () => {
+		const { status } = this.props;
+		this.props.updateCurrentReviewsQuery( this.props.site.ID, { search: '', status } );
+	}
+
+	render() {
+		const { translate, site, status } = this.props;
+		let currentSelection = translate( 'Pending' );
+
+		if ( 'approved' === status ) {
```

Comment:
It is possible that someone might change the string to translate in the markup below but fail to update it here in this conditional. Please consider defining constants above for each of the translated strings and using them both here in this conditional and in the markup below.

Reply:
This is the status returned by the API and is not translated. I could still define constants for these if that helps make it clearer.